packages/venia-concept/webpack.config.js

```diff
@@ -74,5 +74,5 @@ module.exports = async env => {
         })
     ];
 
-    return config;
+    return [clientConfig, serviceWorkerConfig];
```

Comment:
If config is given as an array, webpack will start parallel child processes to build them. Webpack compilation is faster now compared to `develop`.

Reply:
Actually this looks like a good place for extensions compilation if that's a thing 🤔